tests/sentry/models/test_projectkey.py

```diff
@@ -71,28 +81,40 @@ def test_is_active(self):
         assert self.model(project=self.project, status=ProjectKeyStatus.ACTIVE).is_active is True
 
     def test_get_dsn(self):
-        key = self.model(project_id=1, public_key="abc", secret_key="xyz")
-        assert key.dsn_private == "http://abc:xyz@testserver/1"
-        assert key.dsn_public == "http://abc@testserver/1"
-        assert key.csp_endpoint == "http://testserver/api/1/csp-report/?sentry_key=abc"
-        assert key.minidump_endpoint == "http://testserver/api/1/minidump/?sentry_key=abc"
-        assert key.unreal_endpoint == "http://testserver/api/1/unreal/abc/"
+        key = self.model(project_id=self.project.id, public_key="abc", secret_key="xyz")
+        assert key.dsn_private == f"http://abc:xyz@testserver/{self.project.id}"
+        assert key.dsn_public == f"http://abc@testserver/{self.project.id}"
+        assert (
+            key.csp_endpoint
+            == f"http://testserver/api/{self.project.id}/csp-report/?sentry_key=abc"
+        )
+        assert (
+            key.minidump_endpoint
+            == f"http://testserver/api/{self.project.id}/minidump/?sentry_key=abc"
+        )
+        assert key.unreal_endpoint == f"http://testserver/api/{self.project.id}/unreal/abc/"
         assert key.js_sdk_loader_cdn_url == "http://testserver/js-sdk-loader/abc.min.js"
 
     def test_get_dsn_org_subdomain(self):
         with self.feature("organizations:org-subdomains"):
-            key = self.model(project_id=1, public_key="abc", secret_key="xyz")
+            key = self.model(project_id=self.project.id, public_key="abc", secret_key="xyz")
             host = f"o{key.project.organization_id}.ingest.testserver"
 
-            assert key.dsn_private == f"http://abc:xyz@{host}/1"
-            assert key.dsn_public == f"http://abc@{host}/1"
-            assert key.csp_endpoint == f"http://{host}/api/1/csp-report/?sentry_key=abc"
-            assert key.minidump_endpoint == f"http://{host}/api/1/minidump/?sentry_key=abc"
-            assert key.unreal_endpoint == f"http://{host}/api/1/unreal/abc/"
+            assert key.dsn_private == f"http://abc:xyz@{host}/{self.project.id}"
+            assert key.dsn_public == f"http://abc@{host}/{self.project.id}"
+            assert (
+                key.csp_endpoint
+                == f"http://{host}/api/{self.project.id}/csp-report/?sentry_key=abc"
+            )
+            assert (
+                key.minidump_endpoint
+                == f"http://{host}/api/{self.project.id}/minidump/?sentry_key=abc"
+            )
+            assert key.unreal_endpoint == f"http://{host}/api/{self.project.id}/unreal/abc/"
 
 
 @mock.patch("sentry.models.projectkey.schedule_invalidate_project_config")
-@pytest.mark.django_db(transaction=True)
+@pytest.mark.django_db(transaction=True, databases="__all__")
```

Comment:
This is something we're going to need to do in many `django_db()` decorators. I'll fix them up in batches as I identify problematic tests.